alarm/utils.py

```diff
@@ -0,0 +1,90 @@
+import json
+import ssl
+
+import websocket
+from sentry_sdk import capture_exception
+from alarm.models import Coin, Candle
+
+
+def get_binance_price_and_analized(currencies, intervals):
+    websocket.enableTrace(False)
+    sockets_urls = [f'wss://stream.binance.com:9443/ws/{currency}@kline_{interval}' for currency, interval in
+                    zip(currencies, intervals)]
+
+    socket_list = []
+    for socket_url in sockets_urls:
+        socket = websocket.create_connection(socket_url, sslopt={'cert_reqs': ssl.CERT_NONE})
+        socket.on_message = feedback_message_from_websocket
+        socket.on_close = close_streaming_message_from_websocket
+        socket_list.append(socket)
```

Comment:
You said that we can create single connection for all coins

Reply:
https://github.com/digital-brother/binance-alarm/pull/5#discussion_r1153692865